### PR TITLE
[ENH] Use REFRESH CONCURRENTLY to improve user experience.

### PR DESCRIPTION
**Pull request by kittiu**
No view lock when refresh materialized view.

### CODE REVIEW

File: materialized_sql_view/model/abstract_materialized_sql_view.py [hunk 1]
```diff
@@ -336,9 +336,12 @@ def create_mat_view(self, cr, sql, view_name, mat_view_name):
                    dict(mat_view_name=mat_view_name,
                         view_name=view_name,
                         ))
+        cr.execute("CREATE UNIQUE INDEX %(mat_view_name)s_index "
+                   "ON %(mat_view_name)s (id)" %
+                   dict(mat_view_name=mat_view_name, ))
 
     def refresh_mat_view(self, cr, view_name, mat_view_name):
-        cr.execute("REFRESH MATERIALIZED VIEW %(mat_view_name)s" %
+        cr.execute("REFRESH MATERIALIZED VIEW CONCURRENTLY %(mat_view_name)s" %
```
Comment (petrus-v) [inline materialized_sql_view/model/abstract_materialized_sql_view.py:344]: Hi,

Thanks for this PR, `CONCURRENTLY` option is available from PG9.4 only not under PG9.3 so I would suggest to create a new class called PG090400 to implement this feature and adapt getInstance method.

File: materialized_sql_view/views/materialized_sql_view.xml [hunk 2]
```diff
@@ -25,6 +25,10 @@
                                 string="Refresh now"
                                 type="object" states="nonexistent,refreshed,aborted"
                                 groups="materialized_sql_view.manager_group"/>
+	                    <button name="refresh_materialized_view"
+	                            string="Force Refresh"
+	                            type="object" states="refreshing"
+	                            groups="materialized_sql_view.manager_group"/>
```
Comment (petrus-v) [inline materialized_sql_view/views/materialized_sql_view.xml:31]: What would be the difference with "Refresh now" button"?

Comment (pabi2) [inline materialized_sql_view/views/materialized_sql_view.xml:31]: Hello, this button will run the refresh process immediately,  "Refresh now" button will only ask the job to run it.

File: materialized_sql_view/model/abstract_materialized_sql_view.py (hunk 1 above)
Comment (pabi2) [inline materialized_sql_view/model/abstract_materialized_sql_view.py:344]: Yes, that would be great. But I don't know how to do it. :P

**Pull request closed without merging**